### ISSUE TITLE
Android 13 QPR beta builds

### ISSUE THREAD

**Issue by anthonykour** [AndreyPavlenko/Fermata#218]
Android Auto version: 8.7.630214
Packages:
Maps: 11.63.0704
Teams: 1416/1.0.0.2023011703
YouTube Music: 5.40.51
Spotify: 8.8.2.434
Google Play Store: 34.1.09-21 [0] [PR] 501915548
Google Play services: 23.02.54 (190400-501707402)
Fermata Auto: 1.9.0
iHeartRadio: 10.23.0
Messages: messages.android_20230117_01_RC00.phone.openbeta_dynamic
VLC: 3.5.4 Beta 1
Google: 14.3.9.29.arm64
Fermata Control: 1.9.0
SoundCloud: 2023.01.16-beta
Android Auto: 8.7.630214-release
Messenger: 393.0.0.6.92
Google Play Books: 2022.12.12.0.0
Discord: 162.9 - Beta

Permissions:
phone: on
contacts: on
location: on
sms: on
microphone: on
calendar: off
call_log: on
nearby_devices: on

Fermata Control shows up
Fermata Auto does not show up

**Comment by aarshps** [comment]
Facing this same issue. ☹️

**Comment by AndreyPavlenko** [comment]
Have you installed with KingInstaller?

**Comment by anthonykour** [comment]
> Have you installed with KingInstaller?

No, where do I find that?

**Comment by AndreyPavlenko** [comment]
https://forum.xda-developers.com/t/fermata-media-player-audio-and-video-player-for-android-auto.4079519/

**Comment by anthonykour** [comment]
Yeah that didn't work either, but the version on the Google Play store works for now. I guess I'll wait for an updated version to appear on AAAD that supports the Android 13 beta.

**Comment by anthonykour** [comment]
![Screenshot_20230122-150951](https://user-images.githubusercontent.com/28590639/213937927-26b0168f-f67b-49ec-b2e8-e4d48158b483.png)
![Screenshot_20230122-150940](https://user-images.githubusercontent.com/28590639/213937928-968d1284-f95e-4ac9-b5af-d32ae12024ba.png)

**Comment by AndreyPavlenko** [comment]
If it doesn't work, then the only solution is this one - https://github.com/AndreyPavlenko/Fermata/discussions/122#discussioncomment-4307775

**Comment by anthonykour** [comment]
> If it doesn't work, then the only solution is this one - [#122 (comment)](https://github.com/AndreyPavlenko/Fermata/discussions/122#discussioncomment-4307775)

This solution won't work for me as I'm unable to root my phone.

**Comment by anthonykour** [comment]
@AndreyPavlenko will there ever be a future build that supports Android 13, on an unrooted device?

**Comment by AndreyPavlenko** [comment]
@anthonykour The problem is not in the app, but in AA - it blocks apps, that use non-official AA API.

**Comment by majinbu85** [comment]
Anche io mi trovo nella stessa situazione. Il problema credo sia priorio la beta di Android perché sul vecchio pixel 2xl eseguendo la medesima procedura vedo sia fermata auto che fermata control 😭

**Comment by schumi2004** [comment]
> @anthonykour The problem is not in the app, but in AA - it blocks apps, that use non-official AA API.

For my interests. Why not using offical AA API?
And will it help if build it with different package name?
OnePlus Nord user running Pixel Experience 13

**Comment by AndreyPavlenko** [comment]
> For my interests. Why not using offical AA API?

The version from Google Play uses the official API. If you only need a music player, just use that version.



> OnePlus Nord user running Pixel Experience 13

I've never had any problems with PE.
